### ISSUE TITLE
sdl_ttf.go: build error with libsdl2-ttf-dev 2.0.18

### ISSUE THREAD

**Issue by kazzmir** [veandco/go-sdl2#521]
I receive this error when building go-sdl2@v0.4.4
```
cgo-gcc-prolog: In function ‘_cgo_f387abf4bfc4_Cfunc_SDL_GetRevisionNumber’:
cgo-gcc-prolog:69:2: warning: ‘SDL_GetRevisionNumber’ is deprecated [-Wdeprecated-declarations]
In file included from /usr/include/SDL2/SDL.h:61,
                 from ../../go/pkg/mod/github.com/veandco/go-sdl2@v0.4.4/sdl/sdl_wrapper.h:5,
                 from ../../go/pkg/mod/github.com/veandco/go-sdl2@v0.4.4/sdl/version.go:3:
/usr/include/SDL2/SDL_version.h:177:44: note: declared here
  177 | extern SDL_DEPRECATED DECLSPEC int SDLCALL SDL_GetRevisionNumber(void);
      |                                            ^~~~~~~~~~~~~~~~~~~~~
# github.com/veandco/go-sdl2/ttf
../../go/pkg/mod/github.com/veandco/go-sdl2@v0.4.4/ttf/sdl_ttf.go:82:27: cannot use _Ctype_int(val) (value of type _Ctype_int) as type _Ctype_SDL_bool in argument to (_Cfunc_TTF_ByteSwappedUNICODE)
```
The first thing is just a warning, but the second seems to be an incompatability with libsdl-ttf. On the system where this error occurs I have libsdl2-ttf-dev 2.0.18. I have another system that works fine with libsdl2-ttf-dev 2.0.15.

**Comment by veeableful** [comment]
Hi @kazzmir, I believe that has been fixed in more recent version such as v0.4.21. Could you try again after updating go-sdl2?

**Comment by kazzmir** [comment]
You are right, 0.4.21 fixes it. Sorry, I did not realize 0.4.21 was released.
